templates/home/dashboard.html

```diff
@@ -314,6 +314,9 @@ <h4 class="text-center mt-3">Volunteers Present</h4>
 
 <!-- Script for Main Dashboard Date Picker -->
 <script type="text/javascript">
+    "{% now 'Y-m-d' as today_date %}"
+    var current_date = "{{today_date}}"
```

Comment:
You can just put `{% now 'Y-m-d' %}` directly in place of `{{today_date}}` here.